can.js

```diff
@@ -37,9 +37,11 @@ export async function blobAdd(blobPath) {
   }
 
   spinner.start('Storing')
-  const digest = await client.capability.blob.add(blob)
-  const cid = Link.create(raw.code, digest)
-  spinner.stopAndPersist({ symbol: '⁂', text: `Stored ${base58btc.encode(digest.bytes)} (${cid})` })
+  const { multihash } = await client.capability.blob.add(blob, {
+    receiptsEndpoint: client._receiptsEndpoint.toString()
```

Comment:
Uses https://github.com/w3s-project/w3up/blob/695af8b7313a5edc75235cc578d333577d991e92/packages/w3up-client/src/base.js#L31 that is set in test context